operator/README.md

```diff
@@ -0,0 +1,201 @@
+# rapidast-operator
+The RapiDAST operator makes it possible to configure RapiDAST scans from your OpenShift or Kubernetes cluster. This allows you to scan your APIs from both inside and outside of your cluster.
+
+## Custom Resource Definitions
+The RapiDAST operator extends the RapiDAST custom resource definition (CRD) 
+
+## Installing the Operator
+
+There are multiple ways to install the operator onto your cluster. Read through, and select the method that best suits your situation.
+
+### Install via Operator Lifecycle Manager (OLM)
+If your cluster has OLM installed, you can use it to install and manage the RapiDAST operator. You can follow the installation instructions [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md). If you're running on an OpenShift cluster, you likely have this installed by default.
+
+#### Add CatalogSource
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rapidast-catalog
+  namespace: rapidast
+spec:
+  sourceType: grpc
+  image: quay.io/redhatproductsecurity/rapidast-operator-catalog:v0.0.1
+```
+
+#### Add Subscription
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rapidast-operator
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: rapidast-operator
+  source: rapidast-catalog
+  sourceNamespace: rapidast
+```
+### Install with operator-sdk
+
+You can install the operator using the Operator SDK CLI. Installation instructions are available [here](https://sdk.operatorframework.io/docs/installation/)
+
+```bash
+operator-sdk run bundle quay.io/redhatproductsecurity/rapidast-operator-bundle:v0.0.1
+```
+
+You can uninstall with
+```bash
+operator-sdk cleanup --delete-all rapidast-operator
```

Comment:
this doesn't remove the CatalogSource, e.g. rapidast-operator-catalog. So add more info on that.